pinot-common/pom.xml

```diff
@@ -190,6 +190,10 @@
       <groupId>org.restlet.jse</groupId>
       <artifactId>org.restlet</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Good idea, although this is out of scope for this PR. I have created PINOT-3192 to track this.